variables.tf

```diff
@@ -1,11 +1,27 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# REQUIRED PARAMETERS
+# These variables must be set when using this module.
+# ---------------------------------------------------------------------------------------------------------------------
+
+variable "region" {
+  description = "If specified, the AWS region this bucket should reside in. (default: region of the callee)"
+  type        = string
+  default     = null
+}
+
```

Comment:
not required.. this module does not have a single required one.... except for the terraform registry reporting it as required... which is stupid.. but we cannot do anything about it..